src/strtab.rs

```diff
@@ -93,7 +93,7 @@ impl<'a> Index<usize> for Strtab<'a> {
     /// **NB**: this will panic if the underlying bytes are not valid utf8, or the offset is invalid
     #[inline(always)]
     fn index(&self, offset: usize) -> &Self::Output {
-        get_str(offset, &self.bytes, self.delim).unwrap()
+        self.get(offset).unwrap().unwrap()
```

Comment:
I thought it was weird to have two code paths to retrieve records from the `Strtab`. `impl Index` can't be the primary path because it can't return a lifetime, so I made it delegate to `get()`.